Makefile

```diff
@@ -14,6 +14,8 @@ endif
 
 all: manager
 
+update-types: generate manifests
```

Comment:
let's change the `generate` target to do both generate the types and also invoke `manifests` target instead of creating a new target.